src/herbivoresAndCarnivores.js

```diff
@@ -1,15 +1,40 @@
 'use strict';
 
 class Animal {
-  // write your code here
+  constructor(name, health = 100) {
+    this.name = name;
+    this.health = health;
+    Animal.alive.push(this);
+  }
+  // static alive = [];
```

Comment:
```suggestion
  // static alive = [];
```
Так, це було б логічне рішення. Виникла певна проблема з лінтером на нашому боці. Але в цілому - рішення зі static - правильне 👍